src/scss/utilities/_grid.scss

```diff
@@ -100,7 +100,7 @@
   flex-wrap: nowrap;
 }
 
-@include bp-suffix(grid--column, $create-between-breakpoints: true) {
+@include bp-suffix(grid--column, $createBetweenBreakpoints: true) {
```

Comment:
Changing the variables from hyphenated to pascal case will annoy the sass linter / codacy